server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -905,6 +909,19 @@ public void close(boolean saveState) throws IOException {
   void initiateClose(boolean saveState) {
     log.trace("initiateClose(saveState={}) {}", saveState, getExtent());
 
+    synchronized (this) {
+      if (closeState == CloseState.OPEN) {
+        closeRequestTime = System.nanoTime();
+      } else if (closeRequestTime != 0) {
```

Comment:
Is it expected that closeRequestTime would be zero when closeState is not OPEN?  If not expected then could do something like the following to catch bugs

```
}else{
   Preconditions.checkState(closeRequestTime != 0);
```

Reply:
Implemented this change in 3fa4f87